gdsfactory/serialization.py

```diff
@@ -24,13 +24,11 @@
 
 def convert_tuples_to_lists(
     data: dict[str, Any] | list[Any] | tuple[Any, ...] | Any,
-) -> dict[str, Any] | list[Any]:
+) -> dict[str, Any] | list[Any] | Any:
     if isinstance(data, dict):
         return {key: convert_tuples_to_lists(value) for key, value in data.items()}
-    elif isinstance(data, list):
+    elif isinstance(data, list | tuple):
```

Comment:
Note this, feels like if a tuple contains a tuple, they tuple elements should be converted to list too